netlify.toml

```diff
@@ -1,16 +1,6 @@
 [build]
   publish = "public"
-  command = "cd themes/docsy && git submodule update -f --init && npm install && cd ../.. && hugo --gc --minify"
+  command = "npm run build:preview"
 
-[context.deploy-preview.environment]
-  HUGO_VERSION = "0.119.0"
-  NODE_VERSION = "18"
-
-[context.production.environment]
-  HUGO_VERSION = "0.119.0"
-  HUGO_ENV = "production"
-  NODE_VERSION = "18"
-
-[context.branch-deploy.environment]
-  HUGO_VERSION = "0.119.0"
-  NODE_VERSION = "18"
\ No newline at end of file
+[context.production]
+  command = "npm run build:production"
```

Comment:
Are you sure that using `context.production` will still work?

I don't have access to the Netelify dashboard, so I can check, but based on Hugo's docs: https://gohugo.io/hosting-and-deployment/hosting-on-netlify/ they seem to suggest that `context.production.environment` is the right heading.

@james-jwu or @zijianjoy probably have access to Netelify?

Reply:
I'm using similar setups for the OTel and gRPC websites, e.g., see https://github.com/open-telemetry/opentelemetry.io/blob/5df8d080fe6997d1ee514b9f50f6880155276e5b/netlify.toml#L1-L6:

```toml
[build]
publish = "public"
command = "npm run seq -- build:preview diff:check"

[context.production]
command = "npm run seq -- build:production diff:check"
```

The `.environment` section is for setting environment variables only, not for setting other fields like `command`.